packages/client/bin/startRpc.ts

```diff
@@ -0,0 +1,193 @@
+import { Server as RPCServer } from 'jayson/promise'
+import { json as jsonParser } from 'body-parser'
+import Express, { Application, Request, Response, NextFunction } from 'express'
+import jwt from 'express-jwt'
+
+import { RPCManager } from '../lib/rpc'
+import EthereumClient from '../lib/client'
+import { inspectParams } from '../lib/util'
+import * as modules from '../lib/rpc/modules'
+
+type RPCArgs = {
+  rpc: boolean
+  rpcaddr: string
+  rpcport: number
+  ws: boolean
+  wsPort: number
+  wsAddr: string
+  rpcEngine: boolean
+  rpcEngineAddr: string
+  rpcEnginePort: number
+  rpcDebug: boolean
+  helprpc: boolean
+  'jwt-secret': string
+}
+
+function createRPCServerListener({
+  rpcport,
+  server,
+  withEngineMiddleware,
+}: {
+  rpcport: number
+  server: RPCServer
+  withEngineMiddleware?: { jwtEngineSecret: string }
+}): Application {
+  const app = Express()
+  app.use(jsonParser())
```

Comment:
Nit: I think that here you can do `app.use(Express.json())` since it's already in the express library (https://expressjs.com/en/4x/api.html) - This way we don't need to add the `body-parser` dependency in our package json